.travis.yml

```diff
@@ -66,15 +66,14 @@ script:
   - bundle exec rails db:schema:load
   - bundle exec rails assets:precompile
   - './cc-test-reporter before-build'
-  - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "0" ]; then yarn test --colors; fi
+  - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "0" ]; then yarn test --colors;yarn e2e:ci fi
```

Comment:
Since I haven't figured out parallelization of e2e tests yet with the cypress-rails gem and knapsack pro, for now, I'm running the e2e tests on the same node as the front-end tests.